thirdparty/vars.sh

```diff
@@ -288,7 +288,7 @@ JEMALLOC_SOURCE="jemalloc-5.2.1"
 JEMALLOC_MD5SUM="3d41fbf006e6ebffd489bdb304d009ae"
 
 # cctz
-CCTZ_DOWNLOAD="https://github.com/google/cctz/archive/v2.3.tar.gz"
+CCTZ_DOWNLOAD="https://codeload.github.com/google/cctz/tar.gz/refs/tags/v2.3"
```

Comment:
```suggestion
CCTZ_DOWNLOAD="https://github.com/google/cctz/archive/refs/tags/v2.3.tar.gz"
```

Reply:
I check again today and I find that the old value is right too .may be the network traffic yesterday,sorry. please close this pr, 3x a lot